lmdeploy/serve/async_engine.py

```diff
@@ -123,6 +124,12 @@ def _build_turbomind(
         assert isinstance(backend_config, TurbomindEngineConfig), 'Please'\
             ' use TurbomindEngineConfig imported from lmdeploy.messages for ' \
             'turbomind backend'
+        from lmdeploy import turbomind as tm
+        self.engine = tm.TurboMind.from_pretrained(
```

Comment:
If it is moved here, the default `session_len` cannot be updated to `backend_config`

Reply:
> If it is moved here, the default `session_len` cannot be updated to `backend_config`

Let me have a look